src/opentok-screen-sharing.js

```diff
@@ -174,10 +174,12 @@
 
       var container = getContainer();
       var properties =
-        _this.localScreenProperties ||
         _this.localScreenProperties ||
         _defaultScreenProperties;
 
+      if (props) {
+        Object.assign(properties, props);
```

Comment:
If you directly assign `props` to `properties` you will permanently overwrite the values in either `_this.localScreenProperties` or `_defaultScreenProperties`.
It's better to merge all the options in a new var so that we can keep the original values in both `_this.localScreenProperties` and `_defaultScreenProperties`.

I'll do this:
```
var properties = Object.assign({},
  _this.localScreenProperties || _defaultScreenProperties, 
  props)
```

Reply:
change requested was done.